### PR TITLE
fix error in app.blade, move @yield page title to head and create main

### PR DESCRIPTION
**Pull request by Giovanni-Cattaneo**
fix my error following your suggestion

### CODE REVIEW

File: resources/views/layouts/app.blade.php [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    <title>Laravel</title>
+    <title>Laravel @yield('page-title')</title>
```
Comment (Arianna-Salvini) [inline resources/views/layouts/app.blade.php]: Could you add a default value?

**Review by Arianna-Salvini — CHANGES_REQUESTED** [review]
Please make requested changes

File: resources/views/layouts/app.blade.php (hunk 1 above)
Comment (Giovanni-Cattaneo) [reply resources/views/layouts/app.blade.php]: ok no problem but you are a terrbile boss

**Review by Arianna-Salvini — APPROVED** [review]
Thank for your work 😁

**Pull request merged**